### ISSUE TITLE
solov2训练问题

### ISSUE THREAD

**Issue by lpj0711** [PaddlePaddle/PaddleDetection#3784]
> [07/26 11:35:50] ppdet.utils.checkpoint INFO: Finish loading model weights: /home/lpj/.cache/paddle/weights/ResNet50_cos_pretrained.pdparams
[07/26 11:35:52] ppdet.engine INFO: Epoch: [0] [0/3] learning_rate: 0.000000 loss_ins: 2.967660 loss_cate: 1.131157 loss: 4.098817 eta: 0:00:43 batch_cost: 1.2100 data_cost: 0.0002 ips: 0.8265 images/s
[07/26 11:35:55] ppdet.utils.checkpoint INFO: Save checkpoint: output/solov2_r50_fpn_1x_my
[07/26 11:35:55] ppdet.engine INFO: Epoch: [1] [0/3] learning_rate: 0.000030 loss_ins: 2.933610 loss_cate: 1.149617 loss: 4.096600 eta: 0:00:18 batch_cost: 0.5675 data_cost: 0.0002 ips: 1.7620 images/s
[07/26 11:35:58] ppdet.utils.checkpoint INFO: Save checkpoint: output/solov2_r50_fpn_1x_my
[07/26 11:35:59] ppdet.engine INFO: Epoch: [2] [0/3] learning_rate: 0.000060 loss_ins: 2.934256 loss_cate: 1.131157 loss: 4.074222 eta: 0:00:14 batch_cost: 0.4768 data_cost: 0.0002 ips: 2.0973 images/s
[07/26 11:36:02] ppdet.utils.checkpoint INFO: Save checkpoint: output/solov2_r50_fpn_1x_my
[07/26 11:36:02] ppdet.engine INFO: Epoch: [3] [0/3] learning_rate: 0.000090 loss_ins: 2.930676 loss_cate: 1.101606 loss: 4.040696 eta: 0:00:11 batch_cost: 0.4412 data_cost: 0.0002 ips: 2.2664 images/s
[07/26 11:36:05] ppdet.utils.checkpoint INFO: Save checkpoint: output/solov2_r50_fpn_1x_my
[07/26 11:36:05] ppdet.engine INFO: Epoch: [4] [0/3] learning_rate: 0.000120 loss_ins: 2.928387 loss_cate: 1.056145 loss: 4.015069 eta: 0:00:10 batch_cost: 0.4214 data_cost: 0.0002 ips: 2.3729 images/s
[07/26 11:36:09] ppdet.utils.checkpoint INFO: Save checkpoint: output/solov2_r50_fpn_1x_my
[07/26 11:36:09] ppdet.engine INFO: Epoch: [5] [0/3] learning_rate: 0.000150 loss_ins: 2.916640 loss_cate: 1.011280 loss: 3.917385 eta: 0:00:08 batch_cost: 0.4088 data_cost: 0.0002 ips: 2.4461 images/s
[07/26 11:36:12] ppdet.utils.checkpoint INFO: Save checkpoint: output/solov2_r50_fpn_1x_my
[07/26 11:36:13] ppdet.engine INFO: Epoch: [6] [0/3] learning_rate: 0.000180 loss_ins: 2.908507 loss_cate: 0.901732 loss: 3.779836 eta: 0:00:07 batch_cost: 0.4008 data_cost: 0.0002 ips: 2.4947 images/s
[07/26 11:36:16] ppdet.utils.checkpoint INFO: Save checkpoint: output/solov2_r50_fpn_1x_my

**Comment by lpj0711** [comment]
请问日志为什么一直是【0/3】，而且一个Epoch就一个模型，训练12个Epoch就12个模型

**Comment by lpj0711** [comment]
yml配置信息：
> metric: table
num_classes: 3

TrainDataset:
  !COCODataSet
    image_dir: images
    anno_path: json/test.json
    dataset_dir: dataset/solov2_data/
    data_fields: ['image', 'gt_bbox', 'gt_class', 'gt_poly', 'is_crowd']

EvalDataset:
  !COCODataSet
    image_dir: images
    anno_path: json/val.json
    dataset_dir: dataset/solov2_data/

TestDataset:
  !ImageFolder
    anno_path: json/val.json

**Comment by lyuwenyu** [comment]
log打印是按照log_iter间隔打印的 
https://github.com/PaddlePaddle/PaddleDetection/blob/release/2.1/configs/runtime.yml#L2

model是按照snapshot_epoch间隔保存的
https://github.com/PaddlePaddle/PaddleDetection/blob/release/2.1/configs/runtime.yml#L4

**Comment by lpj0711** [comment]
> log打印是按照log_iter间隔打印的
> https://github.com/PaddlePaddle/PaddleDetection/blob/release/2.1/configs/runtime.yml#L2
> 
> model是按照snapshot_epoch间隔保存的
> https://github.com/PaddlePaddle/PaddleDetection/blob/release/2.1/configs/runtime.yml#L4

多谢大佬，可以训练了
